src/components/Company/CompanyName/CorrectNameOptions.vue

```diff
@@ -67,23 +80,28 @@ export default class CorrectNameOptions extends Vue {
   // local properties
   private displayedOptions: Array<CorrectNameOptionIF> = []
   private panel = null as number
-  private submit = false
+  private formType = null
+  private currentFormType = null
```

Comment:
What type are these?

Sorry for being strict/picky but I think we should use types as much as possible to better catch errors during development.

Here, the type cannot be inferred from the initial value, so you should specify it. (In TS with Angular you get warned when specifying a type that is already inferred, so I now leave it out in those situations.)

My basic guideline: I should be able to hover over any property and get some useful type information. Only use 'any' sparingly.

Reply:
No problem at all! 

Updated 👍